cylc/flow/scripts/install.py

```diff
@@ -32,25 +29,35 @@
 A .service directory will also be created and used for server authentication
 files at run time.
 
+If the argument REG is used, Cylc will search for the workflow in the list of
+directories given by `global.cylc[install]source dirs`, and install the first
+match. Otherwise, the workflow in the current working directory, or the one
+specified by --directory, will be installed.
```

Comment:
```suggestion
specified by ``--directory``, will be installed.
```

Reply:
This doesn't get parsed as markup, I do hope we will do one day... but in the mean time most scripts use `"`.